scheduler/cloudformation/cloudformation.go

```diff
@@ -208,6 +208,10 @@ func (s *Scheduler) submit(ctx context.Context, tx *sql.Tx, app *scheduler.App,
 		return err
 	}
 
+	if err := scheduler.Publish(ss, fmt.Sprintf("Created cloudformation template: %v", *t.URL)); err != nil {
```

Comment:
yea i was thinking the same thing initially but also didn't want to just swallow errors. maybe we just log these?